test/controllers/organization_controller_test.exs

```diff
@@ -0,0 +1,109 @@
+defmodule CodeCorps.OrganizationControllerTest do
+  use CodeCorps.ConnCase
+
+  alias CodeCorps.Organization
+  alias CodeCorps.Repo
+  alias CodeCorps.SluggedRoute
+
+  @valid_attrs %{description: "Build a better future.", name: "Code Corps"}
+  @invalid_attrs %{}
+
+  setup do
+    conn =
```

Comment:
This seems like something that might be common between tests, so maybe we should think about abstracting this to its own module. Maybe a test helpers module of some sort.